### ISSUE TITLE
Possible DST issue?

### ISSUE THREAD

**Issue by tijs** [celery/celery#1604]
I just noticed a crontab task setup to run at 12:00 today for some reason ran at at 
both 11:00 and then again at 12:00. I've checked the server date and time and it does seem to have made the switch to winter time last night so i'm not sure if this was a fluke or really related to the DST but i don't see any other reason for it to run the task twice...

My crontab setup, for this particular task, looks like this:

````
from celery.schedules import crontab

CELERYBEAT_SCHEDULE = {
    'event_suggestions': {
        'task': 'events.event_suggestions',
        'schedule': crontab(hour=12, minute=0, day_of_week=1)
    }
}
````
Versions are:
celery==3.0.24
django-celery==3.0.23

**Comment by auvipy** [comment]
Is this a django specific issue?

**Comment by rpkilby** [comment]
We also ran into this issue during the last DST changeover. We have a daily task that executes at 5:00pm, however it was triggered at both 4:00pm and 5:00 pm. I'm wondering if we'll see similar behavior when DST starts in March. 

Using celery version 3.1.18

Django settings roughly look like the following:
```py
USE_TZ = True

TIME_ZONE = 'America/New_York'

CELERY_TIMEZONE = 'America/New_York'

# probably not relevant?
CELERY_ENABLE_UTC = True

CELERYBEAT_SCHEDULE = {
    'send_reminder_emails': {
        'task': 'app.tasks.send_reminder_emails',
        'schedule': crontab(hour=17, minute=0, day_of_week=[1, 2, 3, 4, 5]),
    },
    # snip
    # ...
}
```

@auvipy 
Maybe, but I would doubt it. In our case, there isn't any Django integration beyond loading settings/discovering tasks from the Django project. django-celery isn't being used, so it's not an issue w/ the database-based scheduler.

**Comment by ask** [comment]
Closing this, as we don't have the resources to complete this task.

**Comment by dibell** [comment]
I just hit this issue too. Caused significant confusion and disruption. How can this issue just be closed?!

**Comment by squfrans** [comment]
so another DST change, bitten again.

in my opinion this is a showstopper bug and if (sadly) the project does not have the resources
at the moment to fix it, it should own up to that **openly** so users have the chance to factor
this in in their workflow/risk management.

what is wrong is not _not fixing_ this bug for lack of resources, but _burrying_ it in a closed bug report.

**Comment by rpkilby** [comment]
> what is wrong is not not fixing this bug for lack of resources, but burrying it in a closed bug report.

I kind of have to agree with this. I understand (although disagree with) the idea of closing issues that cannot be tackled. However, by closing the issue and removing bug labels, these issues cannot be tracked. There is no way for potential users to audit the health of a package, understand its caveats etc... 

A few ideas:
- Create a new label to indicate the lack of resources such as "help wanted"
- Create a separate issue to track these issues.

**Comment by thedrow** [comment]
I'm going to reopen this for now. But if it won't be resolved by 5.0 it won't be resolved at all.

**Comment by vbarbaresi** [comment]
Bug should now be solved on master https://github.com/celery/celery/pull/4403
We'll see on the next DST!
